### DISCUSSION TITLE
When i test api. I am not getting proper response

### DISCUSSION THREAD

**Discussion by deangilmoreremix** [shuding/nextra#3155] [General]
And In documentation they are not provide any information regarding error message

I am getting error "something went wrong"

When trying to use the api, we are getting the above error message 

![image (12)](https://github.com/user-attachments/assets/0e068c06-73c6-4cba-81b1-83a4c7fece6e)

**Comment by deangilmoreremix** [comment]
my developer opened a similar case and it was closed without being answered

**Comment by dimaMachina** [comment]
https://github.com/shuding/nextra/issues/3153#issuecomment-2321724776
